Source/events/TouchEvent.swift

```diff
@@ -6,9 +6,34 @@
 //
 //
 
+typealias NodeTouch = (Node, CGPoint)
+
+public enum Relativity {
+    case parent
+    case scene
+}
+
 public struct TouchPoint {
     public let id: Int
-    public let location: Point
+    @available(*, deprecated) public let location: Point // absolute location
+    private let absoluteLocation: Point
```

Comment:
Let's avoid this variable in favor of `location` or make `location` computable to avoid storing unnecessary variable.